autoload/denops_shared_server/runtray.vim

```diff
@@ -0,0 +1,50 @@
+let s:file = expand('<sfile>:p')
+let s:name = 'denops-shared-server'
+let s:local_app_data = isdirectory($LOCALAPPDATA) ? '$LOCALAPPDATA' : '~\AppData\Local'
+let s:config_file = expand(printf('%s\denops\%s.json', s:local_app_data, s:name))
+let s:script_file = expand(printf('%s\denops\%s.ps1', s:local_app_data, s:name))
+let s:config_template_file = printf('%s\runtray.template.json', fnamemodify(s:file, ':h'))
+let s:ps_cmd = 'powershell -NoLogo -NoProfile -ExecutionPolicy RemoteSigned -Command'
+let s:script_download_url = 'https://raw.githubusercontent.com/Milly/runtray-ps/v1.0.0/runtray.ps1'
+
+function! denops_shared_server#runtray#install(options) abort
+  let content = denops_shared_server#_render(readfile(s:config_template_file, 'b'), {
+        \ 'deno': escape(a:options.deno, '\'),
+        \ 'script': escape(a:options.script, '\'),
+        \ 'hostname': a:options.hostname,
+        \ 'port': a:options.port,
+        \})
+  call denops#util#info(printf('create the configuration file `%s`', s:config_file))
+  call mkdir(fnamemodify(s:config_file, ':h'), 'p')
+  call writefile(content, s:config_file, 'b')
+
+  call denops#util#info(printf('download the script `%s`', s:script_file))
+  call denops_shared_server#_system(
+        \ printf('%s Invoke-WebRequest -URI "%s" -OutFile "%s";Unblock-File "%s"',
+        \   s:ps_cmd, s:script_download_url, s:script_file, s:script_file),
+        \ { 'error_message': 'Failed to download the script' })
```

Comment:
This part should be extracted into an internal function (e.g. `denops_shared_server#runtray#_download(...)`) so that we (who know the internal code of `runtray.vim`) can test the functionality manually.

Reply:
This is just my favor but I prefer indentation like

```vim
call denops_shared_server#_system(
      \ printf(
      \  '%s Invoke-WebRequest ...',
      \  s:ps_cmd,
      \  s:script_download_url,
      \  ...
      \ ),
      \ {'error_message': 'Failed to download the script'},
      \)
```

Reply:
Removed `denops_shared_server#_system()` and use `system()`.